django/contrib/auth/forms.py

```diff
@@ -193,19 +179,39 @@ def create_usable_password_field(help_text=usable_password_help_text):
             help_text=help_text,
         )
 
+    @sensitive_variables("password1", "password2")
```

Comment:
I think the sensitive variables marking is not covered by a test

Reply:
Good point, I have added tests for both admin forms which use `SetUnusablePasswordMixin`.